pilot/pkg/model/context.go

```diff
@@ -306,6 +306,12 @@ type Proxy struct {
 	// IPMode of proxy.
 	ipMode IPMode
 
+	// wildCardIPs contains the wildcard host IPs, including IPv4 and IPv6
+	wildCardIPs []string
+
+	// localHostIPs contains the wildcard host IPs, including IPv4 and IPv6
```

Comment:
nit: localHostIPs contains the local host IPs,

Reply:
done!

Reply:
done!